doc/_sphinx/theme/flames.css

```diff
@@ -602,7 +606,7 @@ pre {
   line-height: 125%;
   margin: 0;
   padding: 10px;
-  position: relative;
+  padding-right: 41px;
```

Comment:
Just want to point out that this change will causes empty space and increases scrollable area at all. Are you sure that's a good idea?

| Before   | After    |
| -------- | -------- |
| ![image](https://user-images.githubusercontent.com/4408379/194298726-66aea3bb-a443-4b30-a206-7d59e529fc15.png) | ![image](https://user-images.githubusercontent.com/4408379/194298563-6c056923-63e0-41a4-b766-5b855daecd97.png) |

Reply:
Yes, I have already mentioned before why I have added the extra padding.

Let's take an example of one line:
<img width="644" alt="Screenshot 2022-10-06 at 16 49 57" src="https://user-images.githubusercontent.com/40348358/194299950-272b2ba3-3e8c-4858-ba39-355ce419db83.png">

We won't be able to look at what is there below the tag (in above).

So, adding the padding will make it look like the below:
<img width="646" alt="Screenshot 2022-10-06 at 16 51 09" src="https://user-images.githubusercontent.com/40348358/194300164-4bf71228-71d1-4879-a879-1f54d2205056.png">

This makes a better readability (like 2nd image)

Reply:
This works well for a one-line code block, but increases scrollable area for a multi-line code block. Wouldn't it have been better to add some opacity for a code block lang?

Reply:
Maybe we could just put it above the box, instead of in the box? I don't think opacity would make it very readable it it is above the code either.

Reply:
Then we should probably replace the padding right to padding top? That makes the tag be inside the box.
But, the issue is that it will add extra space on top for all boxes (which will happen even when moving the tag outside).

I feel padding right would be a good option as most won't be scrolling to the complete right and even if they scroll, it doesn't look so odd from what I feel

Reply:
I was thinking without any padding, but putting it outside of the box with a negative margin or such.

Reply:
You mean it like this?

<img width="642" alt="Screenshot 2022-10-06 at 17 12 17" src="https://user-images.githubusercontent.com/40348358/194303944-1180ebc4-4821-4b99-bdbe-d6ee6a282c99.png">

Reply:
Exactly

Reply:
NVM.. I'll add a negative margin top outside 😅